packages/types/src/Metadata/MetadataVersioned.ts

```diff
@@ -20,7 +20,10 @@ import v2ToV1 from './v2/toV1';
 import v3ToV2 from './v3/toV2';
 import v4ToV3 from './v4/toV3';
 
-class MetadataEnum extends EnumType<Null | MetadataV1 | MetadataV2 | MetadataV3 | MetadataV4> {
+class MetadataDeprectated extends Null {
+}
+
+class MetadataEnum extends EnumType<MetadataDeprectated | MetadataV1 | MetadataV2 | MetadataV3 | MetadataV4> {
```

Comment:
So here we're removing MetadataV0, but we can still convert asV0, which is weird

Reply:
Ummm... the change was _only_ supposed to be changing `Null` -> `MetadataDeprecated`